.github/workflows/reusable-run-go-acceptance-test.yml

```diff
@@ -0,0 +1,40 @@
+name: test-run-go-tests-for-path
```

Comment:
I think Ryan suggested this earlier, would be good to rename this workflow file to match the naming pattern here `test-run-go-acceptance-tests` ?

Reply:
I just used Ryan's suggestion verbatim here. I think it's a little more clear that this workflow will only run acceptance tests per the path provided as an input.

Reply:
My mistake, I didn't know we were talking about filenames. I have changed them to be more like the existing ones made by you.